src/@types/NFTFactory.ts

```diff
@@ -21,3 +21,12 @@ export interface NftCreateData {
   transferable: boolean
   owner: string
 }
+
+export interface AccessListData {
+  name: string
+  symbol: string
+  tokenURI: string[]
```

Comment:
why do we need this?

Reply:
it's a typing as `NftCreateData` is used when creating a NFT.